content/docs/security/local-ratelimit.md

```diff
@@ -261,8 +255,154 @@ Instead of applying local rate limiting to a particular route, you can also appl
    local_rate_limited      
    ```
 
-5. Remove the resources that you created in this guide. 
+## Disable rate limiting for a route {#disable-route}
+
+Sometimes, you might want to disable rate limiting for a route. For example, you might have system critical routes that should be accessible even under high traffic conditions, such as a health check or admin endpoints. You can exclude a route from rate limiting by setting the `rateLimit.local` to `{}` in the TrafficPolicy. 
+
+1. Create a Gateway-level TrafficPolicy to enforce local rate limiting on all routes. For more information, refer to the [Gateway configuration](#gateway).
+
+   ```yaml
+   kubectl apply -f- <<EOF
+   apiVersion: gateway.kgateway.dev/v1alpha1
+   kind: TrafficPolicy
+   metadata:
+     name: local-ratelimit
+     namespace: {{< reuse "docs/snippets/ns-system.md" >}}
+   spec:
+     targetRefs: 
+     - group: gateway.networking.k8s.io
+       kind: Gateway
+       name: http
+     rateLimit:
+       local:
+         tokenBucket:
+           maxTokens: 1
+           tokensPerFill: 1
+           fillInterval: 100s
+   EOF
+   ```
+
+2. Create an HTTPRoute for the route that you want to exclude from rate limiting, such as `/anything` on the `httpbin` app.
```

Comment:
maybe say something like: Note that because no TrafficPolicy applies to this HTTPRoute, the Gateway-level rate limiting policy is enforced for the `/anything` route.